client/dashboard/dashboard-charts/block.js

```diff
@@ -25,8 +37,28 @@ class ChartBlock extends Component {
 		}
 
 		return (
-			<Fragment>
-				<Card className="woocommerce-dashboard__chart-block" title={ charts[ 0 ].label }>
+			<div
+				role="presentation"
+				className="woocommerce-dashboard__chart-block-wrapper"
+				onClick={ this.handleChartClick }
+			>
+				<Card
+					className="woocommerce-dashboard__chart-block"
+					title={ charts[ 0 ].label }
+					onClick={ this.handleChartClick }
+				>
+					<a
+						className="woocommerce-dashboard__chart-block screen-reader-text"
+						href={ getAdminLink(
+							'admin.php?page=wc-admin#/analytics/' +
+								charts[ 0 ].endpoint +
+								'?chart=' +
+								charts[ 0 ].key
+						) }
+					>
+						{ /* translators: appended to end of chart type for report link */
+						charts[ 0 ].label + __( ' Report', 'wc-admin' ) }
```

Comment:
You can use sprintf & placeholders to translate this correctly, or just use the chart label directly.

```js
import { __, sprintf } from '@wordpress/i18n';
…
/* translators: %s is the chart type */
sprintf( __( '%s Report', 'wc-admin' ), charts[ 0 ].label );
```

Reply:
Much better way of handling this- thanks!